src/main.rs

```diff
@@ -1,69 +1,41 @@
 // SPDX-FileCopyrightText: 2025 blinry <mail@blinry.org>
```

Comment:
Especially after refactoring, feel free to add yourself as a co-author in both files (in a second line like this, according to the [REUSE specification](https://reuse.software/)). :)